toolchain/gcc/patches/5.4.0/931-fix-mips-use-FP-instruction-on-none-soft-float-targets.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/libffi/src/mips/o32.S b/libffi/src/mips/o32.S
```

Comment:
For more details, please visit below link:

[libffi #264](https://github.com/libffi/libffi/pull/264/commits/b74761126a209e779d0d23a74fe15d09ea69d72a)